src/compiler/checker.ts

```diff
@@ -2289,18 +2289,18 @@ module ts {
             return links.type;
         }
 
-        function getSetAccessorTypeAnnotationNode(accessor: AccessorDeclaration): TypeNode | LiteralExpression {

```

Comment:
I'm not certain.  It's allowed by the grammar.  But i'm not sure it has real meaning.  Regardless, it doesn't need ot be explicitly stated like so.